src/xmldocvisitor.cpp

```diff
@@ -774,7 +778,17 @@ void XmlDocVisitor::visitPre(DocHtmlTable *t)
 {
   if (m_hide) return;
   m_t << "<table rows=\"" << t->numRows()
-      << "\" cols=\"" << t->numColumns() << "\">" ;
+      << "\" cols=\"" << t->numColumns() << "\"" ;
+  HtmlAttribListIterator li(t->attribs());
+  HtmlAttrib* opt;
+  for (li.toFirst(); (opt = li.current()); ++li)
+  {
+    if (opt->name=="width")
+    {
+      m_t << " " << opt->name << "=\"" << opt->value << "\"";
+    }
+  }
+  m_t << ">";
```

Comment:
In the HTML implementation we see that there is the usage of `htmlAttribsToString` and in this implementation the choice, now, has been made to program it for every part.
What is the reason for not using `htmlAttribsToString` or an equivalent version to write to the XML file?

Reply:
I tried to be consistent to the existing code in xmldocvisitor.cpp and keep the change minimal. For example  `XmlDocVisitor::visitPre(DocHtmlCell)` also carefully selects just a few attributes. As far as I see, `htmlAttribsToString` is not used anywhere in xmldocvisitor.cpp so far.

From my perspective it would probably make sense to use `htmlAttribsToString`, that is to pass through all html attributes like the html exporter does. But this would apply to many other tags as well then. I see `htmlAttribsToString` being used 37 times in htmldocvisitor.cpp.